packages/components/tests/dummy/app/templates/components/dropdown.hbs

```diff
@@ -79,424 +130,549 @@
     </dd>
     <dt>icon <code>string</code></dt>
     <dd>
-      <p>Any valid flight-icon name. If none is defined,
-        <span class="dummy-code default">user</span>
-        is set by default</p>
+      <p>Acceptable value: any Flight icon name.</p>
+      <p>Default: <span class="default">"user"</span></p>
```

Comment:
```suggestion
```
[issue] since the default was removed in a PR linked to this one, it should not be listed in the documentation.

Reply:
removed after the conflict resolution